src/components/Groups/Filters.tsx

```diff
@@ -0,0 +1,189 @@
+import React, { FC, useState } from "react"
+import { View } from "react-native"
+import { OutlinedButton } from "./OutlinedButton"
+import { StyleSheet } from "react-native"
+import { ModalSelection, SelectTile } from "components/Settings"
+import { getNameFromMode, ValidModalType } from "utils/groups"
+
+export interface FiltersProps {
+    year: string
+    setYear: (value: string) => void
+    course: string
+    setCourse: (value: string) => void
+    type: string
+    setType: (value: string) => void
+    platform: string
+    setPlatform: (value: string) => void
+    forceSearch: () => void
```

Comment:
al posto di avere un prop + setter per ogni elemento magari ha senso spostare lo state qui dentro e avere un unico callback tipo `onFiltersChange` che restituisce l'intero oggetto, in più cambierei il `forceSearch` a `onReset`, e poi esternamente puoi avere il tuo
```ts
onReset={() => searchGroups()}
```
molto più esplicativo, ma probabilmente nemmeno serve dato che il reset triggererebbe già la modifica dei filtri

Reply:
Si ok non dovrebbe servire